judo-jsl-fullstack-karaf-project-template-application/src/main/resources/docker/pom.xml.hbs

```diff
@@ -0,0 +1,534 @@
+                                </configuration>
+
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <!-- 3. rename unpacked assembly to be able to pack friendly named directory inside docker image -->
+                    <plugin>
+                        <groupId>com.coderplus.maven.plugins</groupId>
+                        <artifactId>copy-rename-maven-plugin</artifactId>
+                        <version>1.0</version>
+                        <executions>
+                            <execution>
+                                <id>rename-jib-image</id>
+                                <phase>prepare-package</phase>
+                                <goals>
+                                    <goal>rename</goal>
+                                </goals>
+                                <configuration>
+                                    <sourceFile>${project.basedir}/target/assembly/{{ lowerCase model.name }}{{ projectPostfix }}-karaf-offline-${project.version}</sourceFile>
+                                    <destinationFile>${project.basedir}/target/assembly/{{ lowerCase model.name }}{{ projectPostfix }}</destinationFile>
+                                </configuration>
+
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+            <dependencies>
+                <dependency>
+                    <groupId>{{ groupId }}</groupId>
+                    <artifactId>{{ lowerCase model.name }}{{ projectPostfix }}-karaf-offline</artifactId>
+                    <type>zip</type>
+                    <version>${project.version}</version>
+                </dependency>
+            </dependencies>
+        </profile>
+
+
+        <profile>
+            <id>create-docker-image-x86_64</id>
+            <activation>
+                <property>
+                    <name>!skipDocker</name>
+                </property>
+				<os><arch>x86_64</arch></os>
+            </activation>
+            <build>
+                <plugins>
+                    <!-- 4. Create image -->
+                    <plugin>
+                        <groupId>com.google.cloud.tools</groupId>
+                        <artifactId>jib-maven-plugin</artifactId>
+                        <version>3.3.2</version>
+                        <executions>
+                            <execution>
+                                <id>build-platform-docker-x86_64</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>buildTar</goal>
+                                </goals>
+
+                                <configuration>
+                                    <from>
+                                        <image>azul/zulu-openjdk-debian:17-latest</image> -->
```

Comment:
Not specifying a version number results in different images being built every single time. Which means we will never be able to tell what version are we compatible with. 

Version should at least be a property.